### PR TITLE
Joint datasets

### PR DESCRIPTION
**Pull request by dvoong**
Added the joint datasets section to the datagroup item view

### CODE REVIEW

File: dataworkspace/dataworkspace/apps/datasets/models.py [hunk 1]
```diff
@@ -330,6 +330,7 @@ class ReferenceDataset(DeletableTimestampedUserModel):
         (SORT_DIR_ASC, 'Ascending'),
         (SORT_DIR_DESC, 'Descending'),
     )
+    is_joint_dataset = models.BooleanField(blank=True, null=True, default=False)
```
Comment (niross) [inline dataworkspace/dataworkspace/apps/datasets/models.py]: Small thing but I don't think we need to set blank or null if we are setting a default here

Comment (dvoong) [reply dataworkspace/dataworkspace/apps/datasets/models.py]: true, updated

Comment (dvoong) [reply dataworkspace/dataworkspace/apps/datasets/models.py]: I've added the migration script now too

File: dataworkspace/dataworkspace/templates/datagroup.html [hunk 2]
```diff
@@ -51,7 +51,30 @@ <h2 class="govuk-heading-s">
                     </h2>
                   {% endif %}
             {% endfor %}
-            {% if datasets and perms.datasets_dataset_add %}
+
+	    {% if joint_datasets or not datasets %}
+	      <h2 class="govuk-heading-l goveuk-!-margin-top-8">Joint datasets</h2>
```
Comment (niross) [inline dataworkspace/dataworkspace/templates/datagroup.html]: I think that should be govuk which will fix the top padding on this heading.

Comment (dvoong) [reply dataworkspace/dataworkspace/templates/datagroup.html]: whoops, fixed it

**Comment by dvoong** [comment]
didn't mean to include that setup mac script into the PR, thanks for catching that

**Comment by dvoong** [comment]
Gove is out! Same for the setup mac script

File: dataworkspace/dataworkspace/templates/datagroup.html [hunk 3]
```diff
@@ -51,7 +51,30 @@ <h2 class="govuk-heading-s">
                     </h2>
                   {% endif %}
             {% endfor %}
-            {% if datasets and perms.datasets_dataset_add %}
+
+	    {% if joint_datasets or not datasets %}
+	      <h2 class="govuk-heading-l govuk-!-margin-top-8">Joint datasets</h2>
```
Comment (niross) [inline dataworkspace/dataworkspace/templates/datagroup.html]: One more small thing, could we just change 'Joint datasets' to 'Available joint datasets' just to match what we have for the other two sections.

Comment (niross) [reply dataworkspace/dataworkspace/templates/datagroup.html]: Actually apologies I've been given new information. Disregard this.

**Pull request merged**